index.html

```diff
@@ -2543,7 +2543,7 @@ <h3><dfn>Writing or pushing content</dfn></h3>
           </li>
           <li>
             If |urlRecord| is failure, [= exception/throw =] a
-            {{"SyntaxError"}} {{DOMException}} and abort these steps.
+            {{TypeError}} {{DOMException}} and abort these steps.
```

Comment:
```suggestion
            {{TypeError}} and abort these steps.
```

Reply:
"and abort these steps" is not technically needed.

Reply:
Remember that DOMExceptions are named, like "AbortError"... while TypeError is an interface :)

Reply:
So when should it be there? "and abort these steps" are everywhere in the steps that throws Exceptions.

Reply:
Yeah, you can probably remove them all. Throwing implies halting (like in normal programming) so it's not needed... however, it's a matter of style, so @kenchris should decide as primary editor.

Reply:
Thanks, ping @kenchris.